shared/common-adapters/input.native.js

```diff
@@ -171,7 +171,8 @@ class Input extends Component<void, Props, State> {
 
     const singlelineStyle = {
       ...commonInputStyle,
-      height: 28,
+      maxHeight: 28,
```

Comment:
:sheep: Why was this necessary instead of specifying `height`? Worth explaining in a comment?